site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -49,6 +49,31 @@ extensions][malicious-mac]).
 
 ## Before you begin {: #prereqs }
 
+### Creating a CRX file {: #create-crx }
```

Comment:
@jpmedley Thanks for 
These instructions are an improvement over the existing "View Permission warnings" instructions under [permission warnings](https://developer.chrome.com/docs/extensions/mv3/permission_warnings/#view_warnings)

I was wondering since the blog post talks about host permission warnings, should we point developers to instructions under the permissions warning doc? 🤔

Reply:
I don't think there's anything there relevant to this particular task and there won't be for some time.

Reply:
Interesting. 

I'd like @dotproto's opinion on this before merging. My vote would go to move these instructions to [View Permission warnings](https://developer.chrome.com/docs/extensions/mv3/permission_warnings/#view_warnings), but I'm okay with whatever the group decides.